patches/eth-query+2.1.2.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/node_modules/eth-query/index.js b/node_modules/eth-query/index.js
+index 13e9f3c..303d703 100644
+--- a/node_modules/eth-query/index.js
```

Comment:
Note that I am still adding a patch for `eth-query`. I'm guessing kumavis still has access to this, but getting a change made might take a while...

Reply:
`EthQuery` is on the way out anyway. We'll be replacing it with ethers soon-ish I hope, maybe after multi-network.